M365/src/DLT365Groupsupgrade.ps1

```diff
@@ -87,8 +87,8 @@ Function Debugdirsync {
     $IsDirSynced = $Distgroup.IsDirSynced
     if ($IsDirSynced -eq $true) {
         $script:Conditionsfailed++
-        Write-Host "Distribution Group can't be upgraded because it's synchronized from on-premises!" -ForegroundColor Red
-        "Distribution Group can't be upgraded because it's synchronized from on-premises!" | Out-File $ExportPath\DlToO365GroupUpgradeChecksREPORT.txt -Append
+        Write-Host "Distribution Group can't be upgraded because it's synchronized from on-premises!`n" -ForegroundColor Red
```

Comment:
Consider creating an output function that writes to both the log and the host, so we don't have duplicate output lines everywhere. That can be a future work item.

Reply:
Thank you for reporting that will work on it on next PR!